backend/src/main/java/com/bakdata/conquery/models/auth/entities/User.java

```diff
@@ -36,6 +37,9 @@
 	private String name;
 	@Getter @Setter @NonNull @NotNull
 	private String label;
+	
+	@Getter @Setter @JsonIgnore
+	private transient boolean hideUserLogout = false;
```

Comment:
Warum ist das pro User?

Reply:
Weil User auf unterschiedlichen wegen authentifiziert werden können. Abhängig davon wie sie authentifiziert wurden, soll der Logout sichtbar sein.

Reply:
Wenn ich mich mit RealmA einlogge (der nicht ausloggen kann), und dann mit RealmB einlogge der ausloggen kann, was steht dann im User?

Reply:
In der Reihenfolge steht dort, dass dir der logout button angezeigt werden soll